src/redux/reducers/compare.ts

```diff
@@ -0,0 +1,385 @@
+import {PayloadAction, TaskAbortError, createSlice, isAnyOf} from '@reduxjs/toolkit';
```

Comment:
I want to split up this file but need to check with Catalin how to structure this feature slice.